modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -85,6 +85,28 @@ public void tearDown() throws Exception {
 		_deleteInstances();
 	}
 
+	@Override
+	@Test
+	public void testGetProcessInstance() throws Exception {
+		Instance instance = randomInstance();
+
+		instance.setSlaResults(
+			new SLAResult[] {
+				_toSLAResult(true, SLAResult.Status.STOPPED),
+				_toSLAResult(true, SLAResult.Status.PAUSED),
+				_toSLAResult(true, SLAResult.Status.RUNNING),
+				_toSLAResult(true, SLAResult.Status.NEW)
```

Comment:
Hi @seltong, please, order the SLAResults by its status in alphabetical order.

Thank you in advance!